src/main/java/hunternif/mc/impl/atlas/client/gui/core/GuiBlinkingImage.java

```diff
@@ -1,19 +1,19 @@
 package hunternif.mc.impl.atlas.client.gui.core;
 
+import org.lwjgl.opengl.GL11;
```

Comment:
I don't think the import is used

Reply:
It's Used Here
GlStateManager.blendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);

Reply:
My bad. nvm.